Sources/ArcGISToolkit/Components/LocationButton/LocationButton.swift

```diff
@@ -0,0 +1,492 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Algorithms
+import ArcGIS
+import CoreLocation
+import SwiftUI
+
+/// A button that allows a user to show their location on a map view.
```

Comment:
```suggestion
/// A button that allows a user to control their location display on a map view.
```

Maybe because the button does more than just hiding/showing the location?